cypress/integration/customizer/hfg/hfg-logo-site-identity.spec.js

```diff
@@ -0,0 +1,172 @@
+const color = '#285078';
+const rgbColor = 'rgb(40, 80, 120)';
+
+describe('Header Builder Logo & Site Identity settings', function () {
+	it('Sets changes and checks them', function () {
+		cy.login();
+		aliasRestRoutes();
+		hideTitleAndTagline();
+		cy.wait('@customizerSave').then((req) => {
+			expect(req.response.body.success).to.be.true;
+			expect(req.status).to.equal(200);
+			cy.wait(2000);
+			checkFrontEndHided();
+		});
+		setupCustomizer();
+		cy.wait('@customizerSave').then((req) => {
+			expect(req.response.body.success).to.be.true;
+			expect(req.status).to.equal(200);
+			cy.wait(2000);
+			checkFrontEnd();
+		});
+
+	});
+});
+
+function aliasRestRoutes() {
+	let home = Cypress.config().baseUrl;
+	cy.server().
+	route('POST', home + '/wp-admin/admin-ajax.php').
+	as('customizerSave');
+}
```

Comment:
This function could be a Cypress command. Check the doc of Custom Commands: docs.cypress.io/api/cypress-api/custom-commands.html